.dockerignore

```diff
@@ -0,0 +1,8 @@
+kube
```

Comment:
Specific? I suppose we could call it "deployment" or something?

Why do you think it's a bad practice to have this stuff in the repo? I've been told it's fine.